java/ql/automodel/test/AutomodelApplicationModeExtraction/Test.java

```diff
@@ -16,56 +16,59 @@
 class Test {
 	public static void main(String[] args) throws Exception {
 		AtomicReference<String> reference = new AtomicReference<>(); // uninteresting (parameterless constructor)
-		reference.set(args[0]); // arg[0] is not a candidate (modeled as value flow step)
-		// ^^^^^^ Argument[this] is a candidate
+		reference.set( // $ sinkModel=set(Object):Argument[this]
+			args[0] // $ negativeSinkExample=set(Object):Argument[0] // modeled as a flow step
+		);  // $ negativeSourceExample=set(Object):ReturnValue // return type is void
 	}
 
 	public static void callSupplier(Supplier<String> supplier) {
-		supplier.get(); // Argument[this] is a sink candidate; the call is a source candidate
+		supplier.get(); // $ sourceModel=get():ReturnValue sinkModel=get():Argument[this]
```

Comment:
```suggestion
supplier.get(); // $ sourceModel=get():ReturnValue sinkModel=get():Argument[this]
```

could we make it so that these are not called "`sourceModel`", but "`sourceModelCandidate`"? as is, this reads a bit if that WAS modelled as a source. Same for `sinkModel`.

Ignore this if it's a significant amount of work.

btw, apologies for the broken whitespace in my suggestion - I can't figure out how to type a TAB character in this input box

Reply:
Great suggestion! Done.